modules/api/php/models/projectdicomsrow.class.inc

```diff
@@ -0,0 +1,123 @@
+<?php declare(strict_types=1);
+/**
+ * PHP Version 7
+ *
+ * @category API
+ * @package  Loris
+ * @author   Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\api\Models;
+
+/**
+ * A ProjectDicomsRow contains values from an image of a project.
+ *
+ * @category API
+ * @package  Loris
+ * @author   Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class ProjectDicomsRow implements \LORIS\Data\DataInstance
+{
+    private $_candid;
+    private $_pscid;
+    private $_entitytype;
+    private $_visitlabel;
+    private $_visitdate;
+    private $_centerid;
+    private $_centername;
+    private $_inserttime;
+    private $_scantype;
+    private $_tarname;
+    private $_dicomarchiveid;
+    private $_source;
+    private $_archive;
+    private $_processid;
+    private $_acquisitionprotocolid;
+    private $_tarchiveseriesid;
+    private $_processprotocolid;
+    /**
+     * Create a new ProjectDicomsRow
+     *
+     * @param array $row An array of image propeties
+     */
+    public function __construct(array $row)
```

Comment:
That constructor is never called.
PDO:statement fetchall instanciate the objects and set their values using `__set`.